### PR TITLE
Stage1 - finishes html / css interface for calculator

### PR DESCRIPTION
**Pull request by mistakeets**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  <meta name="description" content="calculator">
+  <meta name="author" content="Keith Oppel">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100" rel="stylesheet">
+  <link type="text/css" rel="stylesheet" href="main.css">
+</head>
+
+<body>
+  <div class="calculator-container">
+    <div class="calculator-display">
+      <span class="display">0</span>
```
Comment (jrobcodes) [inline index.html:17]: According to the spec, the class name should be namespaced under `calculator`, i.e. `calculator-display`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    </div>
+    <div class="calculator-row">
+      <div class="calculator-button"><button value="AC">AC</button></div>
+      <div class="calculator-button"><button value="±">+/-</button></div>
+      <div class="calculator-button"><button value="%">%</button></div>
+      <div class="calculator-opbutton"><button value="/" style="color:#ffffff">÷</button></div>
```
Comment (jrobcodes) [inline index.html]: Avoid the use of inline styles - these should be applied via the stylesheet.

File: main.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  font-size: 1em;
+  padding: 12px;
+}
+
+button:active :focus {
+  outline: none;
+  border: none;
+}
+
+.calculator-button {
```
Comment (jrobcodes) [inline main.css]: There are a lot of the same rules in both `.calculator-button`, and `.calculator-opbutton`.  Consider applying both styles to the elements that are operators, and specializing the `.calculator-opbutton` to only apply the styles that differ from the standard button.  As well, consider removing the abbreviation in the class name, for clarity: `calculator-operator`.

**Review by jrobcodes — COMMENTED** [review]
Consider omitting `normalize.css` from your project, and serving it instead via a [CDN](https://cdnjs.com/libraries/normalize).

File: index.html (hunk 1 above)
Comment (mistakeets) [reply index.html:17]: Whoops! Must have been "in the zone" and forgot about the naming convention. Perhaps I will rename the display span to calculator-screentext or something similar. Good catch! Is it okay that I am calling the calculator itself calculator-container ?

File: main.css (hunk 3 above)
Comment (mistakeets) [reply main.css]: Thanks for the feedback; I will clean this up before merging

File: index.html (hunk 2 above)
Comment (mistakeets) [reply index.html]: Awww, you don't like my hackey trick... hahah. 😆 I remember doing this as I couldn't get the class to apply the white color to the text on the operator buttons. Perhaps I will ask for help from other people working on the calculator and coach que it if multiple people are having the same issue 😃

**Pull request merged**